kitsune/search/documents.py

```diff
@@ -38,12 +40,6 @@ class WikiDocument(SumoDocument):
     class Index:
         pass
 
-    @classmethod
-    @property
-    def update_document(cls):
```

Comment:
This change will make the `index_object` to fail. If I recall correctly it operates on class types and not on instantiated objects.

Reply:
Why are we making these changes?

Reply:
NOTE: @akatsoulas and I resolved this during a video discussion. My addition of `update_document` as a class-level boolean attribute does the same thing in a simpler way.